website/content/sidebar-examples.json

```diff
@@ -23,6 +23,7 @@
       "tutorials/hello-cube",
       "tutorials/lighting",
       "tutorials/hello-instancing",
+      "tutorials/hello-gltf",
```

Comment:
I think only one of these (the js version probably) is used, the other one should be deleted, a "red herring".